workers/send_email/worker/main.py

```diff
@@ -8,7 +8,7 @@
 
 logging.basicConfig()
 log = logging.getLogger('conductor-workers.send_email')
-log.setLevel(logging.INFO)
```

Comment:
Any change we can make it configured from ENV ? (defaulting to INFO)

Reply:
sure